src/redux/reducers/cellSets/cellSetsUpdateProperty.js

```diff
@@ -1,13 +1,14 @@
+/* eslint-disable no-param-reassign */
 import _ from 'lodash';
+import produce from 'immer';
 
-const cellSetsUpdateProperty = (state, action) => {
-  const { key, dataUpdated } = action.payload;
+import initialState from 'redux/reducers/cellSets/initialState';
 
-  const newState = _.cloneDeep(state);
+const cellSetsUpdateProperty = produce((draft, action) => {
+  const { cellSetKey, dataUpdated } = action.payload;
 
-  _.merge(newState.properties[key], dataUpdated);
-
-  return newState;
-};
+  // Adding original because jest fails otherwise, but in reality it works also without original
```

Comment:
What do you mean by this comment?

Reply:
Removed the reason for this comment but not the comment :P

Reply:
I was trying out some stuff and at a certain point used `original` from immerjs, later I removed it